cths_roles_and_permissions/cths_roles_and_permissions.info

```diff
@@ -0,0 +1,96 @@
+name = CTHS Roles and Permissions
+core = 7.x
+package = Features
+dependencies[] = features
+dependencies[] = islandora
+dependencies[] = islandora_backup
```

Comment:
Is the dependency on this and pathauto something that should be here? Not mentioned in the permissions spreadsheet and they are not in the environment as far as I can tell.

Reply:
Removed - they were originally in the spec but taken out